tee-worker/omni-executor/pumpx/src/pumpx_api/mod.rs

```diff
@@ -66,6 +66,34 @@ impl PumpxApi {
 		})
 	}
 
+	pub async fn verify_google_code(
+		&self,
+		access_token: &str,
+		google_code: String,
+	) -> Result<VerifyGoogleCodeResponse, Error> {
+		let endpoint = format!("{}/v3/account/verify_google_code", self.base_url);
+		let response = self
+			.http_client
+			.post(&endpoint)
+			.bearer_auth(access_token)
```

Comment:
What about `X-Language`? API doc says it's mandatory field

Reply:
We have a default header in the http_client config. but let me pass it as an argument as well